src/foundry/foundry.js/pixi/containers/placeableObjects/token.d.ts

```diff
@@ -369,19 +309,11 @@ declare global {
      */
     checkCollision(destination: Point): boolean;
 
-    /** @override */
-    clone(): Token;
-
-    /** @override */
-    protected _onControl({
-      releaseOthers,
-      updateSight,
-      pan
-    }?: PlaceableObject.ControlOptions & {
-      releaseOthers?: boolean;
-      updateSight?: boolean;
-      pan?: boolean;
-    }): void;
+    /**
+     * @param releaseOthers - (default: `true`)
+     * @param pan           - (default: `false`)
+     */
+    protected _onControl({ releaseOthers, pan }?: { releaseOthers?: boolean; pan?: boolean }): void;
 
     /** @override */
     protected _onRelease({ updateSight }?: PlaceableObject.ReleaseOptions & { updateSight?: boolean }): void;
```

Comment:
Is `updateSight` still necessary here? I can't find any mention of it in foundry.js